### ISSUE TITLE
[Bug]: Restore logging for Package Source Mapping still uses "Package namespace" terminology

### ISSUE THREAD

**Issue by chrisraygill** [NuGet/Home#11306]
### NuGet Product Used

dotnet.exe

### Product Version

dotnet.exe 6.0.100-rc.2.21505.57

### Worked before?

No

### Impact

It bothers me. A fix would be nice

### Repro Steps & Context

1. Create a new project
2. Add a couple test packages
3. Configure a relevant source mapping
4. Run `dotnet restore -f -v d`

![image](https://user-images.githubusercontent.com/15097183/136990835-c4a2f5c1-0b37-44e5-ac1d-13136e1ae7de.png)

### Verbose Logs

```shell
Restoring packages for .NETCoreApp,Version=v6.0...
         Package namespace matches found for package ID 'PackageSourceMappingDemo' are: 'nuget.org,private source'.
         Package namespace matches found for package ID 'AutoMapper' are: 'nuget.org'.
         Package namespace match not found for package ID 'Microsoft.CSharp'.
         Package namespace match not found for package ID 'System.Reflection.Emit'.
         Package namespace matches found for package ID 'Newtonsoft.Json' are: 'nuget.org'.
         Resolving conflicts for net6.0...
```

### Expected behavior

```Package pattern matches found for package ID `AutoMapper` are: `nuget.org` ```

**Comment by nkolev92** [comment]
The build is https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=5264936&view=results.
Insertion issue is https://github.com/NuGet/Client.Engineering/issues/1224

We did insert into the SDK, but they only merged the GA PR, not the RC2 one.

https://github.com/dotnet/sdk/pull/21778
https://github.com/dotnet/sdk/pull/21779#issuecomment-935083080

Arturo tried to engage to get it merged for RC.2 but that didn't happen.

This issue should be fixed in the GA builds of the SDK when they flow into VS.

**Comment by nkolev92** [comment]
The link issue got merged 4 days ago, but apparently has not made it to VS yet.

**Comment by nkolev92** [comment]
Last insetion from SDK into VS was [Thursday](https://devdiv.visualstudio.com/DevDiv/_git/VS/pullrequest/356281). Nothing [pending](https://devdiv.visualstudio.com/DevDiv/_git/VS/pullrequests?_a=active&targetRefName=refs/heads/rel/d17.0&createdBy=122d5278-3e55-4868-9d40-1e28c2515fc4) right now.

**Comment by chrisraygill** [comment]
Inserted and fixed in dotnet version 6.0.100-rtm.21512.4 👍

![image](https://user-images.githubusercontent.com/15097183/136994892-6da177c2-93ba-4115-a310-1ac502887745.png)
